src/bun.js/javascript.zig

```diff
@@ -924,19 +914,19 @@ pub const VirtualMachine = struct {
             this.hide_bun_stackframes = false;
         }
 
-        if (map.map.fetchSwapRemove("NODE_CHANNEL_FD")) |kv| {
-            const mode = if (map.map.fetchSwapRemove("NODE_CHANNEL_SERIALIZATION_MODE")) |mode_kv|
-                IPC.Mode.fromString(mode_kv.value.value) orelse .json
+        if (map.map.get("NODE_CHANNEL_FD")) |kv| {
+            const mode = if (map.map.get("NODE_CHANNEL_SERIALIZATION_MODE")) |mode_kv|
```

Comment:
We should preserve this environment variable? What happens if you use worker_threads inside of an IPC child.